data/data/bootstrap/systemd/units/systemd-journal-gatewayd.service.d/certs.conf

```diff
@@ -0,0 +1,6 @@
+[Service]
+ExecStart=
+ExecStart=/usr/lib/systemd/systemd-journal-gatewayd \
+  --key=/opt/openshift/tls/journal-gatewayd-ca.key \
```

Comment:
since we are using the same set of cert/key pair for server,client-authn and client-authz can we drop the `-ca`